Insta/Insta/ProfileViewController/ProfileItem.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ProfileItem.swift
+//  Insta
+//
+//  Created by Анна Сычева on 21.10.2022.
+//
+
+/// Enum для типов ячеек
+enum ProfileItem {
```

Comment:
```suggestion
enum ProfileItem {
```
название не соответсвует описанию